CourseManager/Views/CourseDetailsView.swift

```diff
@@ -0,0 +1,147 @@
+//
+//  CourseDetailsView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-10-03.
+//
+
+import SwiftUI
+
+struct CourseDetailsView: View {
+    var courseID: String
+    @StateObject var viewModel = CourseDetailsViewModel()
+    @State var showingAddCourseSheet: Bool = false
+    @State var isOpeningCourse = false
+    var view = ContentView()
+
+    var body: some View {
+        Form {
+            courseNameSection
+            courseLinkSection
+            if viewModel.course.gitHubURL != "" {
+                gitHubLinkSection
+            }
+            statusSection
+            if viewModel.course.status == Status.complete {
```

Comment:
Don't need to write `Status.`. It is enough to write `.complete`

Reply:
updated